### PR TITLE
deploy on avalanche and bnb

### PR DESCRIPTION
**Pull request by wilsoncusack**

### CODE REVIEW

**Review by shreck101 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for shreck101 @ 2024-06-14 13:11:24 UTC
User must have write permissions to review

**Review by shreck101 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for shreck101 @ 2024-06-14 13:14:20 UTC
User must have write permissions to review

**Review by shreck101 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for shreck101 @ 2024-06-14 15:56:29 UTC
User must have write permissions to review

**Review by shreck101 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for shreck101 @ 2024-06-15 17:56:59 UTC
User must have write permissions to review

**Review by Johnson1s — COMMENTED** [review]
*

**Review by DatOneJuice — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for DatOneJuice @ 2024-07-29 02:04:11 UTC
User must have write permissions to review

**Comment by cb-heimdall** [comment]
Review Error for Lkhmom85 @ 2024-08-16 03:12:44 UTC
User must have write permissions to review

**Review by jakeshelly — COMMENTED** [review]
Ok

**Review by Chief1934 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for Chief1934 @ 2024-10-01 07:33:16 UTC
User must have write permissions to review

File: broadcast/DeployFactory.s.sol/56/run-latest.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      "to": "0x914d7fec6aac8cd542e72bca78b30650d45643d7",
+      "contractAddress": null
+    }
+  ],
+  "libraries": [],
+  "pending": [],
+  "returns": {},
+  "timestamp": 1715180342,
+  "chain": 56,
+  "commit": "39d5c7c"
```
Comment (Barrywhyte22) [inline broadcast/DeployFactory.s.sol/56/run-latest.json:52]: 52

File: script/DeployFactory.s.sol [hunk 2]
```diff
@@ -20,7 +20,7 @@ contract DeployFactoryScript is Script {
         assert(implementation == EXPECTED_IMPLEMENTATION);
         address factory = SafeSingletonDeployer.broadcastDeploy({
             creationCode: type(CoinbaseSmartWalletFactory).creationCode,
```
Comment (Barrywhyte22) [inline script/DeployFactory.s.sol:22]: CoinbaseSmartWalletFactory

**Review by shreck101 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for shreck101 @ 2024-12-14 12:40:24 UTC
User must have write permissions to review

**Comment by Aaron800** [comment]
Hopefully I can work with it.. Coinbase needs an update

**Comment by DatOneJuice** [comment]
Yes it does & I can’t seem to get anything to work correctly. I need assistance with this mess!

**Pull request merged**